udfs/community/cw_regexp_instr_2.sqlx

```diff
@@ -15,7 +15,10 @@ config { hasOutput: true }
  * limitations under the License.
  */
 
-CREATE OR REPLACE FUNCTION ${self()}(haystack STRING, needle STRING) RETURNS INT64 AS (
+CREATE OR REPLACE FUNCTION ${self()}(haystack STRING, needle STRING) RETURNS INT64 
+OPTIONS (
+    description="""Implements regexp_instr/2 (haystack, needle)."""
```

Comment:
Is that possible to provide a little bit meaningful description so anyone can identify basic use case ? 

```Text
This function aims to find the starting position of the first occurrence of a "needle" string within a "haystack" string.
```